recipe/meta.yaml

```diff
@@ -7,43 +7,51 @@ package:
 source:
   - url: https://github.com/bodono/scs-python/archive/refs/tags/{{ version }}.tar.gz
     sha256: bb509507d857906eb95d17d5824138e48e7c75d1a692d0aa7256d899c7e703c5
+    patches:
+      - gh84.patch
```

Comment:
This was merged upstream. MKL was supported in legacy setuptools build system, but not meson. This PR adds support for MKL in the new meson build system used in the feedstock here.